src/node_report.cc

```diff
@@ -1163,6 +1165,16 @@ void PrintNativeStack(std::ostream& out) {
   out << "\n==== Native Stack Trace ========================================================\n\n";
   out << "Native stack trace not supported on AIX\n";
 }
+#elif (defined(__linux__) && !defined(__GLIBC__))
+/*******************************************************************************
+ * Function to print a native stack backtrace - Alpine Linux etc
+ *
+ ******************************************************************************/
+void PrintNativeStack(std::ostream& out) {
+  out << "\n================================================================================";
+  out << "\n==== Native Stack Trace ========================================================\n\n";
+  out << "Native stack trace not supported on this Linux platform\n";
```

Comment:
Possibly message should say that backtrace isn't supported without GLIBC. Alpine users are likely to recognise that type of issue and users on other Linux's that aren't based on GLIBC might be able to work out what to fix too.